_projects/extend-linter-integration.md

```diff
@@ -93,12 +93,12 @@ difficult to debug things like:
 E           AssertionError: 0 == 0 : The local bear 'CheckstyleBear' yields no result although it should.
 E           Running bear CheckstyleBear...
 E           Running 'java -jar /home/ubuntu/.local/share/coala-bears/CheckstyleBear/checkstyle.jar -c /home/ubuntu/
-			.local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/
-			CheckstyleGood.java'
+              .local/share/coala-bears/CheckstyleBear/google.xml /home/ubuntu/coala-bears/tests/java/test_files/
+            CheckstyleGood.java'
```

Comment:
four space indent

Reply:
still not done.  Read this page: https://github.com/coala/projects/pull/678/files to find the context of my comment. Then it makes sense!

Reply:
Still not modified.